src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -96,6 +96,7 @@ export default function Overlay() {
           if (timeoutRef.current) clearTimeout(timeoutRef.current);
           timeoutRef.current = setTimeout(() => {
             setVisibleOption(undefined);
+            setCommandAmbassador(undefined);
```

Comment:
This fix will only work when the user doesn't interact with the popup triggered by a mod. If the user does immediately interact, this timer gets cancelled and so this reset will never happen.

We probably need a useEffect that immediately sets `commandAmbassador` back to `undefined` when it is any other value? Or maybe a `setImmediate` that does this reset?